guides/notebooks/native-navigation.livemd

```diff
@@ -0,0 +1,377 @@
+# Native Navigation
+
+```elixir
+Mix.install(
+  [
+    {:kino_live_view_native, "0.2.0"}
+  ],
+  config: [
+    live_view_native: [plugins: [LiveViewNative.SwiftUI]],
+    live_view_native_stylesheet: [parsers: [swiftui: LiveViewNative.SwiftUI.RulesParser]],
+    phoenix_template: [
+      format_encoders: [
+        swiftui: Phoenix.HTML.Engine
+      ]
+    ]
+  ]
+)
+
+KinoLiveViewNative.start([])
+```
+
+## Overview
+
+This guide will teach you how to create multi-page applications using LiveView Native. We will cover navigation patterns specific to native applications and how to reuse the existing navigation patterns available in LiveView. 
+
+Before diving in, you should have a basic understanding of navigation in LiveView. You should be familiar with the [redirect/2](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#redirect/2), [push_patch/2](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#push_patch/2) and [push_navigate/2](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#push_navigate/2) functions, which are used to trigger navigation from within a LiveView. Additionally, you should know how to define routes in the router using the [live/4](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.Router.html#live/4) macro.
+
+## NavigationStack
+
+By default, LiveView Native applications are wrapped in a [NavigationStack](https://developer.apple.com/documentation/swiftui/navigationstack) view. Each page is "stacked" on each other, like a list of pages. To see this in action, we'll walk through an example of viewing the LiveView Native Application's source code.
```

Comment:
I don't think this is default, it requires adding `<NavigationStack>` to the root layout. Unless you mean it is default in this notebook.

Reply:
Isn't it the default behavior? We don't add it in the `kino_live_view_native` project, so I assumed it was being injected automatically.

Reply:
Just looked into this: in `0.2.x`, a default root layout is provided if you don't have one. This defaults to wrapping in a `<NavigationStack>`.
But if you have your own root layout, you'll have to add the element to it.

Reply:
I'm modifying the reading to explain it.